src/arti.rs

```diff
@@ -185,6 +197,17 @@ mod tests {
 
         let tempdir = TempDir::new("tor-cache").expect("create temp dir");
 
+        let consensus = include_str!("test-data/consensus.txt");
+        let microdescriptors = include_str!("test-data/microdescriptors.txt");
+
+        let consensus_path = tempdir.path().join("consensus.txt");
+        let mut consensus_file = File::create(consensus_path).expect("create temp consensus file");
+        write!(consensus_file, "{}", consensus).expect("write temp consensus file");
+
+        let microdescriptors_path = tempdir.path().join("microdescriptors.txt");
+        let mut microdescriptors_file = File::create(microdescriptors_path).expect("create temp microdescriptors file");
+        write!(microdescriptors_file, "{}", microdescriptors).expect("write temp microdescriptors file");
```

Comment:
It would be easier to use `std::fs::copy`.

Reply:
That's true, but do we have a way to be certain where the original files will be relatively to the working directory from where the test are run?

Reply:
You can use `env!("CARGO_MANIFEST_DIR")` to have a path to the root of the crate (only set at build time).
Check https://doc.rust-lang.org/cargo/reference/environment-variables.html to find some more.

Reply:
We can do that! O.O